### ISSUE TITLE
Feil ved registrering av nytt medlem

### ISSUE THREAD

**Issue by donv** [Romerike-Jujutsu-Klubb/rjjk_web#240]

**Comment by donv** [comment]
@saxofonia Fikk du feilen da du registrerte "Christopher Honningsvåg" ?

**Comment by saxofonia** [comment]
Ja, og jeg fant ut at han allerede lå der. Sikkert derfor jeg fikk feil?


SRR

fre. 21. jan. 2022, 19:18 skrev Uwe Kubosch ***@***.***>:

**Comment by donv** [comment]
Ja, vi kan registrere foreldre uten fødselsdato, men fødseldato er påkrevd for medlemmer.

Vi fikser det for Christopher, men jeg skal se på registreringsdialogen slik at det ikke skal oppstå igjen.
